db/version_set.h

```diff
@@ -629,8 +629,10 @@ class Version {
   // and return true. Otherwise, return false.
   bool Unref();
 
-  // Add all files listed in the current version to *live.
-  void AddLiveFiles(std::vector<FileDescriptor>* live);
+  // Add all files listed in the current version to *live_table_files and
+  // *live_blob_files.
+  void AddLiveFiles(std::vector<uint64_t>* live_table_files,
```

Comment:
I am a little concerned about returning only file numbers. If we should need more information in the future, we would have to change the signature again.

Reply:
My thought process was that we seem to go to great lengths to make `AddLiveFiles` efficient and release the mutex as soon as possible (e.g. deduplication is done by the caller), yet miss the obvious low-hanging fruit of only copying the data that is actually needed by clients. We can keep it as-is if you see a use for `FileDescriptor`s though; let me know.

Reply:
Matter of taste I guess but I also like the consistency of using the same type. :)

Reply:
SGTM.